requirements.txt

```diff
@@ -1,4 +1,4 @@
-beautifulsoup4==4.9.0
+brotli==1.0.7
```

Comment:
Is `brotli` a package used to enable 'br' encoding?

Reply:
Yes, but requests module already implements Brotli internally, so this line is unnecessary. I will remove it.

Reply:
>but requests module already implements Brotli internally, so this line is unnecessary. I will remove it.

Sorry, this is  a factual error. 
It turns out that if the brotli module is not installed, the compressed data will not decompress properly. 
https://github.com/pusaitou/mikochiku_alarm/issues/48